src/service/poc_mobile.proto

```diff
@@ -106,22 +86,31 @@ enum share_validity {
   share_validity_not_operational = 5;
 }
 
-message moving_avg {
+message speedtest_avg {
```

Comment:
Can we add a timestamp of when the Speed Test average calculation happened?

Reply:
Sure